### PR TITLE
[13.0] [MIG] purchase_line_procurement_group

### PR DESCRIPTION
**Pull request by phuctranfxvn**

### CODE REVIEW

**Comment by rousseldenis** [comment]
@phuctranerp Thanks for your contribution. In the tests, can you maybe replace all the context changes by just one ?

You can do that in setupclass: 

```python
cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

**Comment by phuctranfxvn** [comment]
> @phuctranerp Thanks for your contribution. In the tests, can you maybe replace all the context changes by just one ?
> 
> You can do that in setupclass:
> 
> ```python
> cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
> ```

Hi @rousseldenis: updated, thanks for your suggestion

**Review by rousseldenis — APPROVED** [review]
Code review

File: purchase_line_procurement_group/i18n/purchase_line_procurement_group.pot [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Translation of Odoo Server.
```
Comment (i-vyshnevska) [inline purchase_line_procurement_group/i18n/purchase_line_procurement_group.pot:1]: remove the file

**Review by i-vyshnevska — APPROVED** [review]
LG

File: purchase_line_procurement_group/i18n/purchase_line_procurement_group.pot (hunk 1 above)
Comment (phuctranfxvn) [reply purchase_line_procurement_group/i18n/purchase_line_procurement_group.pot:1]: @i-vyshnevska: this file defines the template for the rest .po files. Why should we remove this file ? I see most modules have this file added. Thanks

**Review by rousseldenis — APPROVED** [review]

**Review by HviorForgeFlow — APPROVED** [review]
Code review :+1:

**Pull request merged**